src/hooks/screen/useChatInfoHook.ts

```diff
@@ -24,15 +33,10 @@ function useChatInfoScreenHook(): UseAnonymousChatInfoScreenHook {
     selectedUser,
     blockModalRef,
     isLoadingInitChat
-  } = useGroupInfo();
-  const {localDb} = useLocalDatabaseHook();
-  const [loadingChannelInfo, setLoadingChannelInfo] = React.useState<boolean>(false);
-  const {isLoadingFetchingChannelDetail, selectedChannel, goBack} = useChatUtilsHook();
-  const [myUserId] = React.useContext(Context).profile;
-  const navigation = useNavigation();
-  const [showPopupBlock, setShowPopupBlock] = React.useState(false);
-  const [channelInfo, setChannelInfo] = React.useState(null);
+  } = useGroupInfo(selectedChannel?.id);
+
   const {signedProfileId, anonProfileId} = useUserAuthHook();
+  const {localDb} = useLocalDatabaseHook();
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `useGroupInfo` hook now accepts a parameter `selectedChannel?.id`. This could potentially break other parts of the code where `useGroupInfo` is called without any parameters. Make sure all instances of `useGroupInfo` are updated to pass this parameter or modify the hook to make the parameter optional.
```diff
- } = useGroupInfo(selectedChannel?.id);
+ } = useGroupInfo(selectedChannel ? selectedChannel.id : undefined);
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->